yarn-project/noir-contracts/src/contracts/private_token_contract/src/main.nr

```diff
@@ -38,7 +37,6 @@ contract PrivateToken {
         let owner_balance = storage.balances.at(owner);
         if (initial_supply != 0) {
             send_note(&mut context, owner_balance, initial_supply, owner);
-            emit_unencrypted_log(&mut context, "Balance set in constructor");
```

Comment:
I don't think we want to do this yet for this contract as it messes up the getting started guides.